examples/js/controls/DragControls.js

```diff
@@ -51,11 +58,12 @@ THREE.DragControls = function ( _objects, _camera, _domElement ) {
 	function onDocumentMouseMove( event ) {
 
 		event.preventDefault();
+                pointer = event.changedTouches ? event.changedTouches[0] : event;
 
 		var rect = _domElement.getBoundingClientRect();
 
-		_mouse.x = ( (event.clientX - rect.left) / rect.width ) * 2 - 1;
-		_mouse.y = - ( (event.clientY - rect.top) / rect.height ) * 2 + 1;
+		_mouse.x = ( (pointer.clientX - rect.left) / rect.width ) * 2 - 1;
+		_mouse.y = - ( (pointer.clientY - rect.top) / rect.height ) * 2 + 1;
```

Comment:
Here to...
```javascript
_mouse.x = ( ( pointer.clientX - rect.left ) / rect.width ) * 2 - 1;
```
Reference https://github.com/mrdoob/three.js/wiki/Mr.doob%27s-Code-Style%E2%84%A2